### ISSUE TITLE
Hyperparameter recommendations for Resnet18 and batch size of 256

### ISSUE THREAD

**Issue by gurdeepmaurya** [facebookresearch/vicreg#21]
Hi,

I can push up to a batch size of 256 on the Resnet18 backbone. 
Could you suggest recommendations on hyperparameters ( Base learning rate, Projector dimensions, similarity/std/cov coefficients, etc.) that would be appropriate for the VICReg framework?

Regards

**Comment by Adrien987k** [comment]
Hi,

The most important hyperparameter is the learning rate, which should be scaled according to the batch size. For smaller batch sizes I recommend increasing the base learning rate. For batch size 256 I used a base learning rate of 0.5 as described in Appendix C.4 of the paper. Projector dimensions and loss coefficients are not as important and can be kept identical.

**Comment by gurdeepmaurya** [comment]
Thanks, Adrien, Wishing you a very happy holiday!!
